transformation/package.json

```diff
@@ -1,51 +1,58 @@
-    "transpile": "tsc",
-    "lint": "./node_modules/.bin/eslint src --ext .ts,.js --fix",
-    "test": "jest"
-  },
-  "dependencies": {
-    "axios": "^0.19.2",
-    "body-parser": "^1.19.0",
-    "cors": "^2.8.5",
-    "express": "^4.17.1",
-    "express-session": "^1.17.1",
-    "keycloak-connect": "^7.0.0",
-    "vm2": "^3.8.3"
-  },
-  "devDependencies": {
-    "@types/cors": "^2.8.6",
-    "@types/express": "^4.17.6",
-    "@types/express-session": "^1.17.0",
-    "@types/jest": "^25.2.3",
-    "@types/keycloak-connect": "^4.5.4",
-    "@types/supertest": "^2.0.9",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
-    "concurrently": "^5.2.0",
-    "eslint": "^6.8.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "jest": "^25.5.4",
-    "ts-jest": "^25.5.1",
-    "tslib": "^1.13.0",
-    "typescript": "^3.9.5"
-  },
-  "homepage": "https://github.com/jvalue/open-data-service",
-  "licenses": [
-    {
-      "type": "GNU Affero General Public License 3",
-      "url": "https://opensource.org/licenses/AGPL-3.0"
-    }
-  ],
-  "repository": "https://github.com/jvalue/open-data-service.git"
+    "name": "open-data-service-transformation",
+    "version": "1.0.0",
+    "description": "TBD",
+    "main": "dist/index.js",
+    "scripts": {
+        "start": "npm run transpile && npm run start:transpiled",
+        "start:transpiled": "node dist/index.js",
+        "transpile": "tsc",
+        "lint": "./node_modules/.bin/eslint src --ext .ts,.js --fix",
+        "test": "jest"
+    },
+    "dependencies": {
+        "amqplib": "^0.5.5",
+        "axios": "^0.19.2",
+        "body-parser": "^1.19.0",
+        "cors": "^2.8.5",
+        "express": "^4.17.1",
+        "express-session": "^1.17.0",
+        "firebase-admin": "^8.10.0",
+        "keycloak-connect": "^7.0.0",
+        "pg": "^8.2.1",
+        "reflect-metadata": "^0.1.13",
+        "typeorm": "^0.2.25",
+        "vm2": "^3.8.3"
+    },
+    "devDependencies": {
+        "@types/node": "^14.0.13",
+        "@types/amqplib": "^0.5.13",
+        "@types/cors": "^2.8.6",
+        "@types/express": "^4.17.6",
+        "@types/express-session": "^1.17.0",
+        "@types/jest": "^25.2.1",
+        "@types/keycloak-connect": "^4.5.2",
+        "@types/supertest": "^2.0.8",
+        "@typescript-eslint/eslint-plugin": "^2.28.0",
+        "@typescript-eslint/parser": "^2.28.0",
+        "concurrently": "^5.1.0",
+        "eslint": "^6.8.0",
+        "eslint-config-standard": "^14.1.1",
+        "eslint-plugin-import": "^2.20.2",
+        "eslint-plugin-node": "^11.1.0",
+        "eslint-plugin-promise": "^4.2.1",
+        "eslint-plugin-standard": "^4.0.1",
+        "jest": "^25.3.0",
+        "ts-jest": "^25.3.1",
+        "tslib": "^1.11.1",
+        "typescript": "^3.8.3"
```

Comment:
Some version got dropped here... you can use `npm update` to insert latest versions automatically